src/ucp/api/ucp.h

```diff
@@ -160,9 +160,9 @@ enum ucp_feature {
     UCP_FEATURE_AM            = UCS_BIT(6),  
 
     /**
-     * Request selection of transports with capability to perform local
-     * operations using a peer's memory buffer and memory handle created
-     * with @ref UCP_MEM_MAP_EXPORTED capability.
+     * Request selection of transports with capability to support mapping a
```

Comment:
I'd remove "Request selection of transports with capability to.." part

Reply:
done